makefile

```diff
@@ -231,8 +231,19 @@ ifeq ($(MFEM_USE_$(1)),YES)
 endif
 endef
 
+# List of deprecated MFEM dependencies, processed below
+MFEM_LEGACY_DEPENDENCIES = OPENMP
+
+# Macro for adding legacy dependencies
+define mfem_add_legacy_dependency
+ifeq ($(MFEM_USE_LEGACY_$(1)),YES)
+   INCFLAGS += $($(1)_OPT)
+   ALL_LIBS += $($(1)_LIB)
+endif
+endef
+
 # Process dependencies
-$(foreach dep,$(MFEM_DEPENDENCIES),$(eval $(call mfem_add_dependency,$(dep))))
+$(foreach dep,$(MFEM_LEGACY_DEPENDENCIES),$(eval $(call mfem_add_legacy_dependency,$(dep))))
```

Comment:
This line should not replace the one with `$(MFEM_DEPENDENCIES)` -- both need to be there. This is seems to be the reason the travis tests failed.

Reply:
Oh I meant to have both. That’s an error I’ll fix